From 23c68de45f6a9b16d74fe83e2620a3ef04efb793 Mon Sep 17 00:00:00 2001
From: Markel <markelsainz@gmail.com>
Date: Thu, 20 Jan 2022 08:46:21 +0100
Subject: [PATCH] My first patch

---
 userprog.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/userprog.c b/userprog.c
index 34619cf..2f1570b 100644
--- a/userprog.c
+++ b/userprog.c
@@ -1,6 +1,6 @@
 #include <stdio.h>
 int main()
 {
-	printf("Hello LWL Team Latest\n");
+	printf("Hello LWL Team Modifications\n");
 	return 0;
 }
-- 
2.25.1

